src/macros.rs

```diff
@@ -256,6 +256,9 @@ macro_rules! intrinsics {
 
         #[cfg(all(target_vendor = "apple", any(target_arch = "x86", target_arch = "x86_64"), not(feature = "mangled-names")))]
         mod $name {
+            #[allow(unused_imports)]
+            use super::*;
```

Comment:
Why are all the new `use super::*` imports required?

Reply:
So that the `CmpResult` is in scope (see the discussion above on `intrinsics!` generating a new module for each function).

Reply:
Thanks, that makes more sense. I think I would prefer `use crate::float::cmp::CmpResult` here, or `-> crate::float::cmp::CmpResult` in `float::cmp` since a glob import in the macro sounds like a recipe for future confusion. It would also be fine to reexport `CmpResult` back up to the top level too so that gets shortened to `crate::CmpResult`.